src/overlay/overlay.scss

```diff
@@ -0,0 +1,376 @@
+// stylelint-disable selector-max-compound-selectors, max-nesting-depth, selector-max-specificity, primer/borders
+// replace with primitive
+$primer-borderRadius-large: 0.75rem;
+
+.Overlay-hidden {
+  display: none !important;
+}
+
+.Overlay-visibilityHidden {
+  height: 0;
+  overflow: hidden;
+  visibility: hidden;
+  opacity: 0;
+}
+
+.Overlay {
+  display: flex;
+  min-width: 192px;
+  flex-direction: column;
+  background-color: var(--color-canvas-default);
+  border-radius: $primer-borderRadius-large;
+  box-shadow: var(--color-overlay-shadow);
+  opacity: 1;
+
+  &.Overlay--height-auto {
+    height: auto;
+  }
+
+  &.Overlay--height-xsmall {
+    height: 192px;
+  }
+
+  &.Overlay--height-small {
+    height: 256px;
+  }
+
+  &.Overlay--height-medium {
+    height: 320px;
+  }
+
+  &.Overlay--height-large {
+    height: 432px;
+  }
+
+  &.Overlay--height-xlarge {
+    height: 600px;
```

Comment:
What happens when the viewport height is smaller than the defined `height` property in this case?

Reply:
It will squish itself to fit with this:

```
max-width: calc(100vw - 64px);
max-height: calc(100vh - 64px);
```

This pretty much only applies to `center` variant. `anchor` variant always (?) uses `auto` height, `full` is fullscreen, and `side` might need some additional finagling..

Reply:
I rewrote sizes to use `min()`. I think its a much better approach!